ansible/group_vars/gelato-ha.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# Dummy variable to avoid error because ansible does not recognize the
+# file as a good configuration file when no variable in it.
+dummy:
+
+
+###########
+# GENERAL #
+###########
+
+#Polulate these IP , Port once the services are installed on K8s.
+gelato_ha_api_ip: API_IP
```

Comment:
Are these API_IP and API_PORT env variables to be exported on the command line?
Need comment or documentation here to explain what these are.
Also API_IP is a very generic name. Since it is specifically for multicloud deployment in HA mode the variable naming can indicate the same.